v2/kafka-to-bigquery/src/main/java/com/google/cloud/teleport/v2/utils/BigQueryAvroUtils.java

```diff
@@ -108,7 +108,7 @@ public class BigQueryAvroUtils {
   private static final DateTimeFormatter DATE_AND_SECONDS_FORMATTER =
       DateTimeFormat.forPattern("yyyy-MM-dd HH:mm:ss").withZoneUTC();
 
-  private static final String kafkaKeyField = "_key";
+  private static final String KAFKA_KEY_FIELD = "_key";
```

Comment:
I think this constant is used in at least 3 different places. Probably makes sense to keep it only in one place. I can't think of a good place though. Maybe a separate `BigQueryConstants` class.

Reply:
done